cmd/launcher/svc_windows.go

```diff
@@ -0,0 +1,103 @@
+// +build windows
+
+package main
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/kit/logutil"
+	"github.com/kolide/launcher/pkg/log/eventlog"
+	"github.com/pkg/errors"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/debug"
+)
+
+// TODO This should be inherited from some setting
+const serviceName = "launcher"
+
+// runWindowsSvc starts launcher as a windows service. This will
+// probably not behave correctly if you start it from the command line.
+func runWindowsSvc(args []string) error {
+	eventLogWriter, err := eventlog.NewWriter(serviceName)
+	if err != nil {
+		return errors.Wrap(err, "create eventlog writer")
+	}
+	defer eventLogWriter.Close()
+
+	logger := eventlog.New(eventLogWriter)
+	level.Debug(logger).Log("msg", "service start requested")
+
+	opts, err := parseOptions(os.Args[2:])
+	if err != nil {
+		level.Info(logger).Log("err", err)
+		os.Exit(1)
+	}
+
+	run := svc.Run
+
+	return run(serviceName, &winSvc{logger: logger, opts: opts})
+}
+
+func runWindowsSvcDebug(args []string) error {
```

Comment:
should this just be a debug flag? Or should we call it `foreground-svc` or something? I feel like there's going to be some confusion around this vs the `-debug` flag.

Reply:
I thought about a flag, but given how the flag options are constructed, that felt awkward. Like, what would that flag do for not-windows? Meanwhile, I renamed to `svc-fg` to make the distinction clear. I think it is an exceedingly uncommon debugging mode